jutut/introduction.rst

```diff
@@ -1,15 +1,113 @@
-.. questionnaire:: math 10
-  :title: Normaali monivalintatehtävä Jutut-luvussa
-  :submissions: 10
+Jutut (officially MOOC-Jutut) is a tool that enables teachers to **collect feedback continuously** during a course, to prompt students to **comment on specific parts of course content**, and to encourage **messaging between students and course staff**. It is an add-on service for A+. Jutut is originally designed for use in large-class courses but it may find uses in smaller-scale teaching as well.
```

Comment:
As discussed in an earlier meeting, should we just start using "Jutut" consistently everywhere, and drop "MOOC-Jutut" entirely, as seems to be done in the rest of the text? If so, I suggest to drop the comment inside parenthesis.

"It is an add-on service for A+" is a short sentence that starts with "It". Pehaps remove, and consider modifiying the first sentence to "Jutut is an add-on service for A+ that enables..."